backend/app/controllers/spree/admin/stock_items_controller.rb

```diff
@@ -1,8 +1,15 @@
 module Spree
   module Admin
-    class StockItemsController < Spree::Admin::BaseController
+    class StockItemsController < ResourceController
       before_action :determine_backorderable, only: :update
 
+      def index
```

Comment:
@tanmay3011 We can remove this method because this controller is inherited from `ResourceController`.